rocAL/rocAL_pybind/amd/rocAL/decoders.py

```diff
@@ -0,0 +1,210 @@
+import amd.rocAL.types as types
```

Comment:
I think, the rocAL name under amd should be all lowercase (rocal).

Reply:
Changed from rocAL to rocal